spec/tail_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'filewatch/tail'
+require 'stud/temporary'
+
+describe FileWatch::Tail do
+
+  let(:file_path) { Stud::Temporary.file.path }
+  let(:sincedb_path) { Stud::Temporary.file.path }
+
+  before :each do
+    Thread.new(subject) { sleep 0.5; subject.quit } # force the subscribe loop to exit
```

Comment:
It would probably be nice to create your own mock out of this construction. Not sure now to name it, but would a class that your instantiate here and does the work of the file, I guess, right?